src/interfaces/ubi/ubiBeneficiaryTransaction.ts

```diff
@@ -0,0 +1,20 @@
+export interface UbiBeneficiaryTransaction {
+    id: number;
+    beneficiary: string;
+    withAddress: string;
+    amount: string;
+    isFromBeneficiary: boolean;
+    tx: string;
+    date: Date;
+    txAt: Date;
+}
+
+export interface UbiBeneficiaryTransactionCreation {
+    beneficiary: string;
+    withAddress: string;
+    amount: string;
+    isFromBeneficiary: boolean;
+    tx: string;
+    date: Date;
+    txAt?: Date;
```

Comment:
remove `date` and make `txAt` mandatory, instead of optional. txAt should not have a default value. It should always be the transaction timestamp.

Reply:
`date` removed and replaced by `txAt`